### PR TITLE
The query condition for active users are unified

### PR DESCRIPTION
**Pull request by annando**
At several occasions we checked for active users - but not always queried the same variables for that. This is now unified.

### CODE REVIEW

File: src/Model/User.php [hunk 1]
```diff
@@ -280,8 +280,7 @@ public static function getActorName(): string
 		// List of possible actor names
 		$possible_accounts = ['friendica', 'actor', 'system', 'internal'];
 		foreach ($possible_accounts as $name) {
-			if (!DBA::exists('user', ['nickname' => $name, 'account_removed' => false, 'account_expired' => false]) &&
-				!DBA::exists('userd', ['username' => $name])) {
+			if (!DBA::exists('user', ['nickname' => $name]) && DBA::exists('userd', ['username' => $name])) {
```
Comment (MrPetovan) [inline src/Model/User.php]: Why are we now positively checking for usernames in `userd`?

Comment (annando) [reply src/Model/User.php]: Because of a mistake 😁

**Review by MrPetovan — APPROVED** [review]

**Comment by MrPetovan** [comment]
Test errors:
```
1) Friendica\Test\src\Model\UserTest::testIdentitiesEmpty
Mockery\Exception\NoMatchingExpectationException: No matching handler found for Mockery_0_Friendica_Database_Database::selectFirst('user', [0 => 'uid', 1 => 'nickname', 2 => 'username', 3 => 'parent-uid'], ['uid' => 1, 'verified' => true, 'blocked' => false, 'account_removed' => false, 'account_expired' => false], []). Either the method was unexpected or its arguments matched no expected argument list for this method



/woodpecker/src/github.com/friendica/friendica/vendor/mockery/mockery/library/Mockery/ExpectationDirector.php:92
/woodpecker/src/github.com/friendica/friendica/src/Database/DBA.php:444
/woodpecker/src/github.com/friendica/friendica/src/Model/User.php:1705
/woodpecker/src/github.com/friendica/friendica/tests/src/Model/UserTest.php:77

2) Friendica\Test\src\Model\UserTest::testIdentitiesAsParent
Mockery\Exception\NoMatchingExpectationException: No matching handler found for Mockery_0_Friendica_Database_Database::selectFirst('user', [0 => 'uid', 1 => 'nickname', 2 => 'username', 3 => 'parent-uid'], ['uid' => 1, 'verified' => true, 'blocked' => false, 'account_removed' => false, 'account_expired' => false], []). Either the method was unexpected or its arguments matched no expected argument list for this method



/woodpecker/src/github.com/friendica/friendica/vendor/mockery/mockery/library/Mockery/ExpectationDirector.php:92
/woodpecker/src/github.com/friendica/friendica/src/Database/DBA.php:444
/woodpecker/src/github.com/friendica/friendica/src/Model/User.php:1705
/woodpecker/src/github.com/friendica/friendica/tests/src/Model/UserTest.php:107

3) Friendica\Test\src\Model\UserTest::testIdentitiesAsChild
Mockery\Exception\NoMatchingExpectationException: No matching handler found for Mockery_0_Friendica_Database_Database::selectFirst('user', [0 => 'uid', 1 => 'nickname', 2 => 'username', 3 => 'parent-uid'], ['uid' => 2, 'verified' => true, 'blocked' => false, 'account_removed' => false, 'account_expired' => false], []). Either the method was unexpected or its arguments matched no expected argument list for this method



/woodpecker/src/github.com/friendica/friendica/vendor/mockery/mockery/library/Mockery/ExpectationDirector.php:92
/woodpecker/src/github.com/friendica/friendica/src/Database/DBA.php:444
/woodpecker/src/github.com/friendica/friendica/src/Model/User.php:1705
/woodpecker/src/github.com/friendica/friendica/tests/src/Model/UserTest.php:150
```
You have to update the conditions in the tests as well.

**Comment by annando** [comment]
Gnarf

**Comment by annando** [comment]
@MrPetovan this time the pipeline hadn't started. Let's hope that the issue was fixed.

**Pull request merged**